test/app.ts

```diff
@@ -1,13 +1,12 @@
-
 import express = require('express');
 import router from './router';
 
 const app: express.Application = express();
 
 // requesthandler
-const requestHandler: express.RequestHandler = (req, res, next) => {
+const requestHandler: express.RequestHandler = (req: express.Request, res: express.Response, next: express.NextFunction) => {
```

Comment:
I've made that change basically because you asked the compiler for `"noImplicitAny": true`